pkg/common/util.go

```diff
@@ -57,7 +58,12 @@ func GetYouTubeService(ctx context.Context, fs *firestore.Client, userID string)
 			return err
 		},
 	)
-	return youtube.New(notifyHook.Client(ctx))
+	client := retryablehttp.NewClient()
+	client.HTTPClient = notifyHook.Client(ctx)
```

Comment:
HTTP requests are logged to stderr, which don't get forwarded to GCP Cloud Logging. This is probably fine.